DoSomething/gambit-conversations
# Rogue voting plans: part 1 [pr]

#### What's this PR do?

This PR executes a Rogue `POST /posts` request after a user has answered all questions that collect voting plan information, creating a text post from the relevant user voting plan fields.

#### How should this be reviewed?

* Send yourself an `askVotingPlanStatus` broadcast (e.g. [`4LJn0P1LeoUauC0CkESYKE`](https://gambit-admin.herokuapp.com/broadcasts/4LJn0P1LeoUauC0CkESYKE))
* Submit a yes answer to begin voting plan data collection
* Upon submitting the method of transportation, verify a new text post has been created for your user

#### Any background context you want to provide?

Coming soon in part 2:

* Unit tests for `helpers.user.createVotingPlan`

* Execute a `GET /posts` request to avoid creating multiple voting plan text posts per TODO

#### Relevant tickets

https://www.pivotaltracker.com/story/show/160824069

#### Checklist
- [x] ENV variables added/updated for new features/bug fixes.
